priv/repo/structure.sql

```diff
@@ -61,6 +62,38 @@ CREATE SEQUENCE public.charlie_cards_id_seq
 ALTER SEQUENCE public.charlie_cards_id_seq OWNED BY public.charlie_cards.id;
 
 
+--
+-- Name: products; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.products (
+    id bigint NOT NULL,
+    name character varying(255) NOT NULL,
```

Comment:
I'd make this a `:text` rather than a 255 string. In fact we should just use :text everywhere and if we want shorter strings we do that in the application, not in the db

Reply:
@napilla You should probably come up with a team agreement regarding these kinds of fields.